src/v2/Apps/Order/Routes/__tests__/Status.jest.tsx

```diff
@@ -124,21 +122,50 @@ describe("Status", () => {
       it("should say confirmed and have message box", async () => {
         const page = await buildPageWithOrder({
           ...OfferOrderWithShippingDetails,
-          ...PaymentDetails,
-          state: "APPROVED",
           displayState: "IN_TRANSIT",
         })
         expect(page.text()).toContain("Your order has shipped")
         expect(page.getMessage()).toBe(1)
       })
+
+      it("should display non linked tracking number if no Url", async () => {
+        const page = await buildPageWithOrder({
+          ...ArtaOrderWithShippingDetailsTrackingNoUrl,
+          displayState: "IN_TRANSIT",
+        })
+        expect(page.text()).toContain("oxa")
+        expect(
+          page.find(Message).find("Message").find("RouterLink").length
+        ).toBe(0)
+      })
+
+      it("should display link to tracking URL if present", async () => {
+        const page = await buildPageWithOrder({
+          ...OfferOrderWithShippingDetails,
```

Comment:
re: refactoring -- this entire app's test suite uses an abstraction ([`createTestEnv`](https://github.com/artsy/force/blob/24735dbbc1b659c303ac9d1d49ae3c6f6367a960/src/v2/DevTools/createTestEnv.tsx)) which in its current form & usage locks us into using:
* these manually constructed mocks (instead of relay test utils auto-mocking)
* enzyme for testing (instead of react-testing-library)

It'd be great, in the future, for us to refactor this abstraction (or eliminate it?) so that we can use the alternatives.

It'd be fun to spike on this some time to see what it would take to swap the order app's tests, maybe we can do that for a future friday some time.

Reply:
(and if it wasn't clear, I agree 100% that refactoring of that level should be something we take on outside of the scope of this long-ago-merged PR :)